Feature/selectcustomer

DevUnderConstruction opened the pull request.

DevUnderConstruction committed:
add BillCard which accepts customers and an onclick on Usercard which renders it

DevUnderConstruction committed:
add SelectCustomer component and map function which renders all customers as select option

DevUnderConstruction committed:
add function to store selected customer data

DevUnderConstruction committed:
add button which is enabled if customer is selected

ufelgen commented on components/BillCard.js:
```diff
@@ -0,0 +1,24 @@
+
+    const selectedCustomerId = Number(event.target.value);
+    const selectedCustomer = customers.find(
+      (customer) => customer.id === Number(event.target.value)
```
```suggestion
      (customer) => customer.id === selectedCustomerId
```

ufelgen reviewed (commented):
Hey Jonas, 
dein Code sieht, soweit ich das überblicken kann, gut aus! ich hab noch einen Vorschlag gemacht, dass du deine Konstante noch einbindest, die du in der einen Funktion vergibst. 
Aber ich hab mal versucht, das in der Preview nachzuvollziehen und ich schaffe es, dass das Dropdown-Menü erscheint, aber wenn ich auf "select" klicke, dann passiert nichts... was soll denn da passieren?

DevUnderConstruction committed:
Update components/BillCard.js

Ja haste recht👍 die variable ist noch vom testen übrig. Ich übernehme deinen Vorschlag

Co-authored-by: Ulrike Felgenhauer <115288914+ufelgen@users.noreply.github.com>
